api/v1/requests/AttendeeRequest.js

```diff
@@ -6,24 +6,20 @@ const extraInfoValidations = {
   info: ['string', 'maxLength:255']
 };
 
-const projectErrorMessage = 'The projects supplied are invalid. Attendees can only create 1 project at most.';
-const projectValidations = {
-  name: ['required', 'string', 'maxLength:100'],
-  description: ['required', 'string', 'maxLength:255'],
-  repo: ['required', 'string', 'maxLength:150'],
-  isSuggestion: ['required', 'boolean']
-}; // NOTE: these are currently not supported
+const requestedCollaboratorValidations = {
+  collaborator: ['required', 'string', 'maxLength:255']
+};
 
-const ecosystemInterestValidations = {
-  ecosystemId: ['required', 'integer']
+const websiteValidations = {
```

Comment:
Did the pre-existing extrainfo validations not do this?

Reply:
They did. I have reverted back to the original validations and named them properly in 7570badb275fb9eabb0d396aa6ce83e6a5ff4fec.